helm/cluster-cloud-director/templates/kubeadmcontrolplane.yaml

```diff
@@ -111,6 +111,12 @@ spec:
       {{- if $.Values.network.staticRoutes }}
       {{- include "staticRoutes" . | nindent 6}}
       {{- end }}
+    ntp:
+      {{- if .Values.ntp.enabled }}
```

Comment:
move this part to a common function and call it in kubeadmcontrolplane and kubeadmconfig spec.